build.json

```diff
@@ -1,35 +1,35 @@
 {
-  "UseSystemDotNetPath": "false",
-  "DotNetFolder": ".dotnet",
-  "DotNetInstallScriptURL": "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0-preview2/scripts/obtain",
-  "DotNetChannel": "preview",
-  "DotNetVersion": "1.0.0-preview2-003131",
-  "BuildToolsFolder": ".tools",
-  "ArtifactsFolder": "artifacts",
-  "TestProjects": {
-    "OmniSharp.MSBuild.Tests": [
-      "netcoreapp1.0"
+    "UseSystemDotNetPath": "false",
```

Comment:
FWIW, it'd be nice if this change didn't update the whitespace in the files, which messes up the diff.